bundles/com.espressif.idf.core/src/com/espressif/idf/core/util/HintsUtil.java

```diff
@@ -0,0 +1,54 @@
+/*******************************************************************************
+ * Copyright 2022-2023 Espressif Systems (Shanghai) PTE LTD. All rights reserved.
+ * Use is subject to license terms.
+ *******************************************************************************/
+package com.espressif.idf.core.util;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.InputStream;
+import java.util.ArrayList;
+import java.util.LinkedHashMap;
+import java.util.List;
+
+import org.yaml.snakeyaml.Yaml;
+
+import com.espressif.idf.core.logging.Logger;
+
+public class HintsUtil
+{
+
+	public static List<String[]> getReHintsList()
+	{
+
+		File hintsYmFile = new File(IDFUtil.getIDFPath() + File.separator + "tools" + File.separator + "idf_py_actions"
+				+ File.separator
+				+ "hints.yml"); //$NON-NLS-1$ //$NON-NLS-2$ //$NON-NLS-3$
```

Comment:
texts and NON-NLS seems to be on the different lines

Reply:
fixed